src/components/hmi_message_handler/src/messagebroker_adapter.cc

```diff
@@ -88,58 +88,85 @@ void MessageBrokerAdapter::processNotification(Json::Value& root) {
 
 void MessageBrokerAdapter::SubscribeTo() {
   LOG4CXX_INFO(logger_, "MessageBrokerAdapter::subscribeTo");
-  MessageBrokerController::subscribeTo("Buttons.OnButtonEvent");
-  MessageBrokerController::subscribeTo("Buttons.OnButtonPress");
-  MessageBrokerController::subscribeTo("UI.OnCommand");
-  MessageBrokerController::subscribeTo("VR.OnCommand");
-  MessageBrokerController::subscribeTo("BasicCommunication.OnReady");
+
+  // BasicCommunication
+  MessageBrokerController::subscribeTo("BasicCommunication.OnAppActivated");
```

Comment:
@dev-gh added only RC-related subscriptions in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/1be4f2b22b538d88b968d4358a69cada7af38911